src/pages/MyProfile.jsx

```diff
@@ -10,9 +11,34 @@ import hide from "../assets/hide-pass.svg";
 import show from "../assets/show-pass.svg";
 import {toast, ToastContainer} from "react-toastify";
 import 'react-toastify/dist/ReactToastify.css';
+import {useNavigate} from "react-router-dom";
+
+
+const StyledPopup = styled(PopupJS)`
+
+        &-content {
+            border-radius: 10px;
+            width: 100%;
+        }
+        @media (min-width: 770px) {
+            &-content {
+                border-radius: 10px;
+                width: 60%;
+            }
+        }
+        
+        @media (min-width: 1180px) {
+            &-content {
+                border-radius: 10px;
+                width: 40%;
+            }
+        }
+    `;

```

Comment:
why are we using vanilla CSS?

Reply:
Like it says in the comment above, it was the only way I could make the popup responsive. If you can find a better way, feel free to change it.